contrib/test/integration/test.yml

```diff
@@ -7,10 +7,13 @@
     regexp: ' go test \"\$'
     state: present
 
-- name: set extra storage options
+- name: Usernamespace integration testing uses updated environment variables
   set_fact:
-    extra_storage_opts: " --storage-opt overlay.override_kernel_check=1"
-  when: ansible_distribution == 'RedHat' or ansible_distribution == 'CentOS'
+    # Keys/Values in userns_int_test_env update int_test_env (from vars.yml)
+    int_test_env: '{{ int_test_env | combine(userns_int_test_env) }}'
```

Comment:
I'm sure there's a simple answer to this, but I've given up looking: won't this *always* add `userns_int_test_env` to `int_test_env`? Shouldn't there be a `when` or conditional or something so it only happens in the `userns` case?

Reply:
Ahh you're right, and this is confusing.  I mistakenly thought the tags would behave like a logical ``and`` instead of ``or``.  I just checked, even with only ``userns`` here, having the ``integration`` tag on the ``import: tests.yml`` task results in the same behavior.  Hmmm...the quick/easy fix seems to be, move this up into ``main``, w/ only the ``userns`` tag on it.  Got a better idea?